ql/src/test/queries/clientnegative/cbo_fallback_skip_with_except.q

```diff
@@ -0,0 +1,2 @@
+set hive.cbo.fallback.strategy=CONSERVATIVE;
+select cast(0 as bigint) = '1' except select cast(1 as bigint) = '1';
```

Comment:
Let's just keep this q test and remove the qualify which is more involved.

Reply:
Removed the one with qualify
https://github.com/apache/hive/pull/5312/commits/1ad217d7bfb4147e265ea6f43914afc293b3cbbb